media/js/mozorg/contribute/signup.js

```diff
@@ -0,0 +1,21 @@
+$(function() {
```

Comment:
As long as the JS is below/after all markup (just above the `</body>` tag), you shouldn't have to worry about it. The DOM will be ready. It does feel uneasy at first, but I've not once seen a problem. #trustfalls